service/src/util/serviceUtils.ts

```diff
@@ -106,3 +106,29 @@ export async function modifyResourcesForClone(artifacts: FhirArtifact[], version
 
   return artifacts;
 }
+
+export function createArtifactComment(
+  type: string,
```

Comment:
This can be stricter (I think) by limiting it to just 'guidance', 'review' and 'documentation'